vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -51,4 +51,16 @@ namespace Microsoft.ReactNative.Bridge
 
     Windows.Foundation.Collections.IMapView<String, IInspectable> ExportedCustomDirectEventTypeConstants { get; };
   };
+
+  [webhosthidden]
+  interface IViewManagerWithChildren
+  {
+    void AddView(Windows.UI.Xaml.FrameworkElement parent, Windows.UI.Xaml.UIElement child, Int64 index);
```

Comment:
>Xaml [](start = 28, length = 4)

I wonder if it is possible to define view managers in a way that Xaml is not used in the API. I.e. to make it more as an implementation detail for a platform.
Or it does not make sense and the API always must rely on Xaml?

Reply:
Technically yes, the even rawer internal API just gets the tag IDs from ReactNative, and we get the corresponding UI element before continuing - we might normalize the external APIs around the tags, but this then we'd have to make sure to expose the APIs to convert them.